ckanext/datatablesview/plugin.py

```diff
@@ -22,6 +22,7 @@ class DataTablesView(p.SingletonPlugin):
     p.implements(p.IConfigurer, inherit=True)
     p.implements(p.IResourceView, inherit=True)
     p.implements(p.IBlueprint)
+    p.implements(p.ITemplateHelpers)
```

Comment:
BTW, instead of `implements(ITemplateHelpers)` and defining `get_helpers` you can use [`@tk.blanket.helpers`](https://docs.ckan.org/en/2.10/extensions/plugins-toolkit.html#ckan.plugins.toolkit.ckan.plugins.toolkit.blanket) class decorator. It will take all public functions defined in the `helpers` module and register them as plugin's helpers.

It's not a mandatory way, but it a nice solution for people, like you, who creates all the functions in the right place:)